### PR TITLE
[IMP] Newly created users get all manager access right

### PR DESCRIPTION
**Pull request by tivisse**
Coming from a bug in web_settings_dashboard. Invited user didn't have any rights
when created from the dashboard, which was leading to an error.

This bug leaded to a new discussion. Better to have basic employee having user
rights for all main applications. For bigger entreprises there is an admin that
will carefully remove extra rights, if necessary. The target is small businesses,
it makes sense that every way to create a user gives the same result.

In conclusion, each new user has a full access to the applications by default

### CODE REVIEW

File: openerp/addons/base/res/res_users.py [hunk 1]
```diff
@@ -262,16 +262,9 @@ def _get_companies(self, cr, uid, context=None):
...
-        result = []
-        try:
-            dummy,group_id = dataobj.get_object_reference(cr, SUPERUSER_ID, 'base', 'group_user')
-            result.append(group_id)
-            dummy,group_id = dataobj.get_object_reference(cr, SUPERUSER_ID, 'base', 'group_partner_manager')
-            result.append(group_id)
-        except ValueError:
-            # If these groups does not exists anymore
-            pass
+        default_user_id = self.pool['ir.model.data'].xmlid_to_res_id(cr, uid, 'base.default_user')
```
Comment (tde-banana-odoo) [inline openerp/addons/base/res/res_users.py]: Maybe use xmlid_to_object ? (ou _record, je sais plus et j'ai la flemme d'aller voir dans le code)

Comment (tde-banana-odoo) [inline openerp/addons/base/res/res_users.py]: Be carefull also that some people could remove this user. The code should not crash, if default user does not exist, the default groups is void.
